server/src/main/scala/services/AirportToCountry.scala

```diff
@@ -0,0 +1,53 @@
+
+package services
+
+import drt.shared.CrunchApi._
+import drt.shared._
+import uk.gov.homeoffice.drt.ports.PortCode
+import uk.gov.homeoffice.drt.redlist.RedListUpdates
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.io.Codec
+import scala.language.postfixOps
+import scala.util.Try
+
+trait AirportToCountryLike {
```

Comment:
This trait `AirportToCountryLike` is only used with `AirportToCountry` so there's no need for it - we can just move its contents into the object